canonical-ols/build.snapcraft.io
# add workflow text and images + snap image

## Done

On the landing page: 

* Added images and text for the workflow page based on the [copy doc](https://docs.google.com/document/d/1ijyI7EMPu0BoyVbB-mPB7gyKnPyloC0Rvx1Oq-yEqKk/edit#)
* Added snap block to the snap section
* Reduced the size of the top illustration per Jamie

## QA

1. go to the landing page
2. see that the top illustration is 920px
3. see that the workflow section is correct compared to the copy doc
4. see there is a snap image in the snap section
